packages/stream-tile-handler/src/__tests__/tile-document.test.ts

```diff
@@ -243,8 +251,8 @@ describe('TileDocumentHandler', () => {
     })
 
     it('creates genesis records deterministically if deterministic:true is specified', async () => {
-        const record1 = await TileDocument.makeGenesis(context.api, RECORDS.genesis.data, { deterministic: true })
-        const record2 = await TileDocument.makeGenesis(context.api, RECORDS.genesis.data, { deterministic: true })
+        const record1 = await TileDocument.makeGenesis(context.api, null, { deterministic: true })
```

Comment:
Nitpick: it would be nice to put *something* into the metadata just so we see that the deterministic documents have something in them.  I'd just add a family and a tag